include/zephyr/bluetooth/mesh/health_srv.h

```diff
@@ -158,7 +158,7 @@ struct bt_mesh_health_srv {
 	struct k_work_delayable attn_timer;
 
 #ifdef CONFIG_BT_MESH_LARGE_COMP_DATA_SRV
-	/** Pointer to the array with Health Test Info Metadata */
+	/** @deprecated { User model->metadata instead } */
```

Comment:
Not sure it is correct to mark this field as deprecated. This makes wrong assumption that users can still use their code (which uses this pointer) for 2 more releases while it actually won't work and they need to change their code.

Reply:
Then what is the alternative?